src/track/cueinfoimporter.cpp

```diff
@@ -0,0 +1,63 @@
+/// cueinfoimporter.cpp
+/// Created 2020-04-24 by Jan Holthuis
+
```

Comment:
This is already in the Git history. Do we really need to add it?

Reply:
I saw it in some files in the mixxx codebase. I can remove it if you want.

Reply:
please do